spec/queries/contributions_by_user_query_spec.rb

```diff
@@ -45,12 +45,15 @@
   end
 
   context '#per_month' do
+    let(:today) { Date.today }
+    let(:last_month) { today.month - 1 }
```

Comment:
Are you sure about it? If the month is January, what will be the `last_month` value?

What you can do is `(today - 1.month).month`, but again, for January, will it generate the expected behavior?

Reply:
https://github.com/Codeminer42/Punchclock/pull/96/commits/91fa81c0d2c9e1b8754d6e4846e01fda60dd299c